docs/ref-index.rst

```diff
@@ -23,6 +23,7 @@ references for background information about indexation.
     .. automethod:: recordlinkage.Index.full
     .. automethod:: recordlinkage.Index.block
     .. automethod:: recordlinkage.Index.sortedneighbourhood
+    .. automethod:: recordlinkage.Index.neighbourhoodblock
```

Comment:
I'm not completely sure on what to do with the documentation on the contrib module. I think I will create a new subsection in the docs. For now, removing these changes will help.